### PR TITLE
Add job template for cloud connector

### PR DESCRIPTION
**Pull request by ehelms**

### CODE REVIEW

**Comment by ehelms** [comment]
This previously lived in foreman_ansible -- https://github.com/theforeman/foreman_ansible/pull/526

**Comment by ShimShtein** [comment]
@ehelms I have fixed the tests. Ca you please rebase this?

**Comment by adamruzicka** [comment]
> Can you please rebase this?

Once more maybe?

**Comment by ehelms** [comment]
Updated

File: app/views/job_templates/cloud_connector.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<%#
+name: Configure Cloud Connector
+snippet: false
+template_inputs:
+- name: satellite_cloud_connector_user
```
Comment (adamruzicka) [inline app/views/job_templates/cloud_connector.erb:5]: The input names were different. Either they need to stay the same or the rename needs to be done here https://github.com/theforeman/foreman_rh_cloud/blob/master/app/services/foreman_rh_cloud/cloud_connector.rb#L19 as well

Comment (ehelms) [reply app/views/job_templates/cloud_connector.erb:5]: Great catch, fixed!

**Pull request merged**